### PR TITLE
[Subnet] add try catch in delete subnet for subnet routing rule deletion if routing table is not exist

### PR DESCRIPTION
**Pull request by yanmo96**
added try catch to 'delete subnet routing rule in route manager'.
From second part of issue #648

### CODE REVIEW

**Review by cj-chung — APPROVED** [review]
LGTM

**Review by xieus — APPROVED** [review]
LGTM

File: services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/controller/SubnetController.java [hunk 1]
```diff
@@ -537,7 +539,11 @@ public ResponseId deleteSubnetState(@PathVariable String projectId, @PathVariabl
             }
 
             // delete subnet routing rule in route manager
-            this.subnetService.deleteSubnetRoutingRuleInRM(projectId, subnetId);
+            try {
+                this.subnetService.deleteSubnetRoutingRuleInRM(projectId, subnetId);
```
Comment (xieus) [inline services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/controller/SubnetController.java:543]: This method was to delete the entire routing table associated with this subnet in RM. Let us rename it to deleteeSubnetRoutingTable.

**Comment by xieus** [comment]
@yanmo96 Could you update this PR? It has been a while.

**Comment by yanmo96** [comment]
Will do, sorry miss the earlier comment.

**Comment by yanmo96** [comment]
Name changed

**Comment by xieus** [comment]
This is a workaround to fix Issue #660

**Review by xieus — APPROVED** [review]
LGTM

**Pull request merged**